app/src/main/java/br/org/cesar/discordtime/stickysessions/navigation/router/Router.java

```diff
@@ -0,0 +1,38 @@
+package br.org.cesar.discordtime.stickysessions.navigation.router;
+
+import android.app.Activity;
+
+import br.org.cesar.discordtime.stickysessions.R;
+import br.org.cesar.discordtime.stickysessions.navigation.exception.InvalidRouteException;
+import br.org.cesar.discordtime.stickysessions.navigation.repository.IRouteRepository;
+import br.org.cesar.discordtime.stickysessions.navigation.wrapper.IContextWrapper;
+
+public class Router implements IRouter {
+
+    private IContextWrapper mContext;
+    private IRouteRepository routeRepository;
+
+    public Router(IRouteRepository routeRepository) {
+        this.routeRepository = routeRepository;
+    }
+
+    @Override
+    public void setContext(IContextWrapper context) {
```

Comment:
Follow the Follow Naming Conventions, you don't need to use **this.context** here (only mContext)